docs/backend/testing/acceptance-testing-payment-requests.md

```diff
@@ -6,6 +6,8 @@ sidebar_position: 3
 
 Link to a video walk-through example of acceptance testing a payment request. [Video in Google Drive](https://drive.google.com/file/d/1yDQAgRxGRGwkiwhUKgaor5C0vMtm6Ek4/view)
 
+EDI 858 Structure with segment significance and source data mapping can be found in [google drive](https://docs.google.com/spreadsheets/d/12vNs6X-jqLIeDTntvFkYUG98S_FOFyWSRNs8LQCeYwg/edit?usp=sharing).
```

Comment:
I recall we do tend to put a lock icon for links to non-public resources

Reply:
@duncan-truss while I'm here, that link to the video above doesn't work for me. Do I lack permissions, or should I remove that line?

Reply:
https://github.com/transcom/mymove-docs/pull/404/commits/39b3c6805ccb5528e6f677f3a90bac5801004b51

Reply:
I opted to move this documentation centrally to its own page in this commit, where it could be more easily linked from other pages (which I'll do next): https://github.com/transcom/mymove-docs/pull/404/commits/3ed67959806cbb69735cb7e360f76037a2d2e85d

Reply:
And that's done in https://github.com/transcom/mymove-docs/pull/404/commits/a573852081ab7e28118472b90091676084310202